framework/test/unit/modules/token/method.spec.ts

```diff
@@ -1225,11 +1230,61 @@ describe('token module', () => {
 	});
 
 	describe('supportAllTokensFromChainID', () => {
-		it('should call support chain', async () => {
-			await expect(
-				method.supportAllTokensFromChainID(methodContext, Buffer.from([1, 2, 3, 4])),
-			).resolves.toBeUndefined();
+		it('should return early if all supported token entries is truthy', async () => {
+			jest.spyOn(method as any, '_isAllTokensSupported').mockResolvedValue(true);
```

Comment:
Overall in the test, instead of stubbing the method, it's better to use real function. I think most of the problem on the source code should be identified by that